doc: samples/synchronization: update README.rst

youndong opened the pull request:
Updated with actual console results output to console. 
Also, when target board is selected as qemu_x86_64, 
it shows the execution results on dual cpu.

Signed-off-by: Youndong Park <youndong.park@gmail.com>

youndong committed:
doc: samples/synchronization: update README.rst

Updated with actual console results output to console. 
Also, when target board is selected as qemu_x86_64, 
it shows the execution results on dual cpu.

Signed-off-by: Youndong Park <youndong.park@gmail.com>

nashif commented:
@youndong  you should use the same PR for updates, do not create new PRs for each modification.

youndong commented:
@nashif, 
Regarding this commit, I'm sorry to make you incovient. Since this is my first commit for zephyr, there was a mistake. I hope your understanding. Thank you.

stephanosio commented on samples/synchronization/README.rst:
```diff
@@ -30,16 +30,44 @@ Sample Output
+   :goals: run
+   :compact:
+
+Sample Output
+=============
```
This seems redundant.

youndong replied:
I have explicitly separated the results for the two target boards.

stephanosio commented on samples/synchronization/README.rst:
```diff
@@ -30,16 +30,44 @@ Sample Output
+
+   <repeats endlessly>
+
+And, if the board was selected qemu_x86_64, this demonstration would 
+show the following results.
```
It would be nice to clarify that the purpose of the first example is to test non-SMP, and the second is to test SMP.

youndong committed:
Merge branch 'zephyrproject-rtos:main' into main

youndong committed:
doc: samples/synchronization: update README.rst

Updated with actual console results output to console.
Also, when target board is selected as qemu_x86_64,
it shows the execution results on dual cpu.

stephanosio reviewed (changes requested):
Please read https://docs.zephyrproject.org/latest/contribute/index.html#contribution-workflow; in particular, about force pushing.

youndong commented:
> Please read https://docs.zephyrproject.org/latest/contribute/index.html#contribution-workflow; in particular, about force pushing.

Thank you, @stephanosio. It will be helpful for me.

youndong committed:
doc: samples/synchronization: update README.rst

Updated with actual console results output to console.
Also, when target board is selected as qemu_x86_64,
it shows the execution results on dual cpu.
